plugins/stateful/src/App.tsx

```diff
@@ -0,0 +1,22 @@
+import '@github/relative-time-element';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import React from 'react';
+
+
+import { Splash } from './components/Splash';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+});
+
+export const App = () => {
+  return (
+    <QueryClientProvider client={queryClient}>
```

Comment:
@duvanmonsa Move both providers `react-query` and `apollo-client` here

Reply:
Apollo-client requires react-query to be ready that's why they can't be together at least for now.